### PR TITLE
tests_node_ext: platform fixes

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
- Tests now pass on n30,n31,n5,n6,n7,n8,n9-I2,n9-I3
- Known image issue with last_reason_time on n8,n9_I2

### CODE REVIEW

File: lib/cisco_node_utils/cmd_ref/show_version.yaml [hunk 1]
```diff
@@ -50,18 +50,19 @@ last_reset_reason:
...
   get_value: "rr_ctime"
 
 system_image:
-  nexus:
-    get_value: "kick_file_name"
+  get_value: "kick_file_name"
+  N5k: &sys_img_isan
+    get_value: "isan_file_name"
+  N6k: *sys_img_isan
+  N7k: *sys_img_isan
```
Comment (robert-w-gries) [inline lib/cisco_node_utils/cmd_ref/show_version.yaml:63]: Can you remind me what the `&` and `*` symbols do here?

File: lib/cisco_node_utils/cmd_ref/system.yaml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 # system
 ---
 resources:
-  nexus:
-    get_command: "show system resources"
-  # ios_xr:
-    # grpc doesn't allow this command
-    # get_command: 'monitor threads iteration 1'
+  # grpc doesn't allow this command
+  # get_command: 'monitor threads iteration 1'
```
Comment (robert-w-gries) [inline lib/cisco_node_utils/cmd_ref/system.yaml:5]: @glennmatthews @chburket Does this comment still apply?

File: lib/cisco_node_utils/cmd_ref/show_version.yaml (hunk 1 above)
Comment (chrisvanheuveln) [inline lib/cisco_node_utils/cmd_ref/show_version.yaml:63]: `&sys_img_isan` is just a tag. `*sys_img_isan` uses the values found at that tag.
It allows us to specify the same value for all three of those platforms.

**Comment by robert-w-gries** [comment]
:+1:

File: lib/cisco_node_utils/cmd_ref/show_version.yaml (hunk 1 above)
Comment (mikewiebe) [inline lib/cisco_node_utils/cmd_ref/show_version.yaml:63]: @robert-w-gries , YAML anchors and aliases: See: https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/README_YAML.md#yaml-anchors-and-aliases

Comment (robert-w-gries) [inline lib/cisco_node_utils/cmd_ref/show_version.yaml:63]: That's neat.  It's sometimes difficult to remember all of the nice features in our YAML :-)

File: lib/cisco_node_utils/cmd_ref/system.yaml (hunk 2 above)
Comment (glennmatthews) [inline lib/cisco_node_utils/cmd_ref/system.yaml:5]: Yes it does. grpc blocks all exec commands that don't start with 'show '.

(Edit) What's the purpose of this change, though - why not leave the file alone?

Comment (chrisvanheuveln) [inline lib/cisco_node_utils/cmd_ref/system.yaml:5]: It was okay before, it's better now. :-)

Kidding... I was troubleshooting one of the tests and just changed it since I prefer the explicit _exclude over the implicit "unsupported". But I'll change it back if you'd prefer.

Comment (glennmatthews) [inline lib/cisco_node_utils/cmd_ref/system.yaml:5]: No biggie, just wondering. Was this needed to get the 'validate_property_excluded' check to work?

Comment (chrisvanheuveln) [inline lib/cisco_node_utils/cmd_ref/system.yaml:5]: Nope. The validate works with both syntax types.
